pydantic_ai_slim/pydantic_ai/common_tools/exa.py

```
@@ -22,24 +26,41 @@
-    from exa_py.api import ContentsOptions, TextContentsOptions
+    from exa_py.api import ContentsOptions, Result, SearchResponse
 
 __all__ = (
+    'ContentType',
     'ExaToolset',
+    'SearchType',
     'exa_search_tool',
     'exa_find_similar_tool',
     'exa_get_contents_tool',
     'exa_answer_tool',
 )
```

Ruff (0.15.21)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)